portfolio/src/main/webapp/script.js

```diff
@@ -32,45 +32,37 @@ function addRandomGreeting() {
 }
 
 
-/**
- * Fetches a random quote from the server and adds it to the DOM.
+/** returns comments directly without promises and adds them to the DOM 
  */
-function getData() {
-  console.log('Fetching data from server.');
-
-  // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
 
-  // When the request is complete, pass the response into handleResponse().
-  responsePromise.then(handleResponse);
+async function getComments() {
+  const response = await fetch('/data');
+  console.log('Fetching data from server.');
+  const comments = await response.json();
+  console.log(comments);
```

Comment:
You might want to delete the extra console.log() statements to print comments and commentsElements (typically used for status updates, error printing, etc.)